CleverSDK/Classes/CLVCleverSDK.h

```diff
@@ -9,9 +9,10 @@
 #ifndef SimpleLogin_CLVCleverSDK_h
 #define SimpleLogin_CLVCleverSDK_h
 
+#define SDK_VERSION @"iOS-0.1.3"
```

Comment:
Can we grab the SDK_VERSION from the Info.plist instead of hardcoding? That way, we can have a single source of truth

Reply:
Seems like this pattern is also frequently used. 👍